CryptoAnalysis/src/main/java/crypto/reporting/CommandLineReporter.java

```diff
@@ -26,10 +30,23 @@
 	private File outputFolder;
 	private List<CrySLRule> rules;
 	private Collection<IAnalysisSeed> objects = new HashSet<>();
+	private static final Logger LOG = LoggerFactory.getLogger(CommandLineReporter.class);
+	/**
+	 * format of CrytoAnalysis report
+	 */
+	private Format reportformat;
 
-	public CommandLineReporter(String string, List<CrySLRule> rules) {
-		this.outputFolder = (string != null ? new File(string) : null);
+	/**
+	 * Creates {@link CommandLineReporter} a constructor with reportDir, format and rules as parameter
+	 * 
+	 * @param reportDir a {@link String} path giving the location of the report directory
+	 * @param format {@link Format} the format of CryptoAnalysis report
+	 * @param rules {@link CrySLRule} the rules with which the project is analyzed
+	 */
+	public CommandLineReporter(String reportDir, Format format, List<CrySLRule> rules) {
```

Comment:
I wonder why the  `CommandLineReporter` is writing stuff to a file. Shouldn't it just print messages to the command line. and a nother reporter class e.g. `FileReporter`?